src/util/dir_utils.py

```diff
@@ -11,6 +11,21 @@ def payment_report_file_path(pymnt_root, pymnt_cycle, nb_failed):
     return os.path.abspath(os.path.join(pymnt_root, PAYMENT_DONE_DIR if nb_failed == 0 else PAYMENT_FAILED_DIR, str(pymnt_cycle) + '.csv'))
 
 
+def get_latest_report_file(payments_root):
+    recent = None
+    if get_successful_payments_dir(payments_root):
+        files = [os.path.splitext(x)[0] for x in os.listdir(get_successful_payments_dir(payments_root))]
+        paid_cycles = []
+        for x in files:
+            try:
+                paid_cycles.append(int(x))
```

Comment:
The way you suggested leads to failure if the target dir contains an unexpected file. Report files are named after cycle numbers e.g. 54.csv, 55.csv... I prefer to leave as is.

Reply:
Is it ok to ignore a file here if it's unexpected? Maybe it would be good to give the user a warning or something. A file extension check would be also beneficial like if file.endswith("csv") then append if you they need to be in csv format.